core/src/main/java/org/geysermc/geyser/translator/protocol/java/JavaLoginTranslator.java

```diff
@@ -130,6 +133,17 @@ public void translate(GeyserSession session, ClientboundLoginPacket packet) {
         }
         session.sendDownstreamPacket(new ServerboundCustomPayloadPacket(register, Constants.PLUGIN_MESSAGE.getBytes(StandardCharsets.UTF_8)));
 
+        Set<NetworkChannel> registeredChannels = session.getNetworkManager().registeredChannels();
+        if (!registeredChannels.isEmpty()) {
+            String channels = registeredChannels
+                    .stream()
+                    .filter(channel -> !channel.isPacket())
+                    .map(channel -> channel.identifier().namespace() + ":" + channel.identifier().path())
+                    .collect(Collectors.joining("\0"));
+
+            session.sendDownstreamPacket(new ServerboundCustomPayloadPacket(register, channels.getBytes(StandardCharsets.UTF_8)));
```

Comment:
Nit: doing this here won't allow sending e.g. custom payloads during the config phase